apps/api/src/app/export/export.service.ts

```diff
@@ -95,7 +99,10 @@ export class ExportService {
                 : SymbolProfile.symbol
           };
         }
-      )
+      ),
+      user: {
+        settings: { currency: userWithSettings.Settings.settings.baseCurrency }
```

Comment:
You can use `userCurrency`, then you don't even have to make a query 🙂

```
settings: { currency: userCurrency }
```

Reply:
Fixed!